### PR TITLE
Added email functionality to load long pipeline

### PR DESCRIPTION
**Pull request by Leon-JS02**
Emails are now sent when the long term pipeline is carried out.

### CODE REVIEW

File: long_term/load_long.py [hunk 1]
```diff
@@ -30,6 +30,45 @@ def get_s3_client():
...
+            },
+            'Body': {
+                'Html': {
+                    'Data': html_body
+                }
+            }
+        }
+    )
+
+
```
Comment (olliejth) [inline long_term/load_long.py:71]: Good idea doing this in the code, could this be changed to a step function in the future?

**Review by olliejth — APPROVED** [review]
Added the email function ver elegantly.

File: long_term/load_long.py (hunk 1 above)
Comment (Leon-JS02) [reply long_term/load_long.py:71]: Good idea for future improvements!

**Review by YakMan101 — APPROVED** [review]
Given time limit applying the emailing functionality in script is good quick fix.

**Pull request merged**